viewer-prototype/src/browser/trace-viewer/components/table-output-component.tsx

```diff
@@ -140,12 +143,21 @@ export class TableOutputComponent extends AbstractOutputComponent<TableOutputPro
         }
 
         columnEntries.forEach(columnHeader => {
+            const typeString: string | null = columnHeader.type;
+            let hiddenColumn = false;
+            if (typeString) {
+                const type: ColumnHeaderType = JSON.parse(typeString);
+                hiddenColumn = type.hidden;
+            }
+
             const id = this.showIndexColumn ? ++columnHeader.id : columnHeader.id;
             colIds.push(id);
             columnsArray.push({
                 headerName: columnHeader.name,
                 field: columnHeader.id.toString(),
-                width: this.props.columnWidth
+                width: this.props.columnWidth,
+                hide: hiddenColumn
```

Comment:
According to the ag-grid, the hide property accepts boolean.
https://www.ag-grid.com/javascript-grid-column-properties/

In our case, the hidden column is set to string after JSON.parse. It's good to set to boolean though it works in this case.